### ISSUE TITLE
Improve releasing CI for the output targets repo

### ISSUE THREAD

**Issue by benelan** [Esri/calcite-components#6117]
### Summary

We currently have a [really rough script](https://github.com/ArcGIS/calcite-components-output-targets/blob/master/support/prepareReactRelease.js) to release Calcite Components React (CCR), which I run after releasing Calcite Components (CC). I want to automate it so that CCR automatically releases after CC does. Including `next` releases, which CCR doesn't have at the moment. 

### Desired Outcome

A quick brainstorm:

#### Option A
- Send a POST request from CC to CCR after releases. 
- Add a GitHub Action in CCR that recieves the request using the [`dispatch_dispatch`](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#repository_dispatch) trigger.
- Determine the type of the CC release (major/minor/patch), and use [`node-semver`](https://github.com/npm/node-semver) to increment the CCR version. Or  [`release-please`](https://github.com/googleapis/release-please) if we want to add a CCR changelog.
- Add a bunch of sanity checks to the new CCR release script to make sure things don't explode

#### Option B
- Monorepo 🚀

**Comment by SergioMartinSanchez** [comment]
Hi @benelan !
I'm recently trying to use my Stenciljs components in remix, but I haven't been able to do it.
But I have checked that due to the processes that you do to export the react wrapper with your components, if they are compatible with serving them in remix.run, following the POC that you did in this [branch](https://github.com/benelan/stencil-remix/tree/fix#stencil-remix) ,
I would like to know if you plan to make public [this configuration](https://github.com/ArcGIS/calcite-components-output-targets/blob/master/support/prepareReactRelease.js) to make react wrappers, as clearly the stenciljs documentation is not suitable for this use case.

Best regards and thanks!

**Comment by benelan** [comment]
Hi @SergioMartinSanchez we are considering transitioning this repo to a monorepo, which would include the react output target. But no promises at this time. We followed stencil's example, so you can use that as a reference for now:
https://github.com/ionic-team/stencil-ds-output-targets/tree/main/packages/react-output-target

The main issue is that the browser's custom element APIs don't exist on the server, so rendering web components there throws errors. The issue isn't specific to Calcite. The workaround is using dynamic imports to force the custom elements to render on the client, but than can be fragile.


Here is a stencil issue with some more info:
https://github.com/ionic-team/stencil-ds-output-targets/issues/226

**Comment by SergioMartinSanchez** [comment]
@benelan  thanks for the quick response!
Yes I already tested the react target and dynamically load the entry point but it fails when I try to load the stencil core

<img width="1166" alt="CleanShot 2023-02-27 at 23 31 56@2x" src="https://user-images.githubusercontent.com/10806524/221700929-1e9dfaef-9b67-4e8e-bc5e-8881389e759c.png">

and when I saw in your repository the [patchESMResolution.ts](https://github.com/Esri/calcite-components/blob/master/support/patchESMResolution.ts) script I thought it was related.

I will keep trying, thanks!
